File: TabelaConjuncao.java
public class TabelaConjuncao {
    public static void main(String[] args) {
        boolean[] valores = {true, false};

        System.out.println("Tabela Verdade - Conjunção\n");
        System.out.println("A\t\tB\t\tA && B");

        for (boolean a : valores) {
            for (boolean b : valores) {
                boolean resultado = a && b;
                System.out.println(a + "\t" + b + "\t" + resultado);
            }
        }
    }
}

File: TabelaConjuncao.py
def main():
    valoresA = []
    valoresB = []

    print("TABELA VERDADE - CONJUNÇÃO")
    print("Digite V para verdadeiro e F para falso.")

    for i in range(4):
        a = input(f"Proposição {i+1} - Valor A (V/F): ").strip().upper()
        b = input(f"Proposição {i+1} - Valor B (V/F): ").strip().upper()

        valoresA.append(a == 'V')
        valoresB.append(b == 'V')

    print("\nTABELA VERDADE - CONJUNÇÃO (AND - and)")
    print("A\tB\tA and B")
    for a, b in zip(valoresA, valoresB):
        print(f"{a}\t{b}\t{a and b}")

if __name__ == "__main__":
    main()
